pkg/cmd/extension/command.go

```diff
@@ -300,7 +300,7 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				Long: heredoc.Doc(`
 					Install a GitHub repository locally as a GitHub CLI extension.
 
-					The repository argument can be specified in "owner/repo" format as well as a full URL.
+					The repository argument can be specified in "OWNER/REPO" format as well as a full URL.
 					The URL format is useful when the repository is not hosted on github.com.
 
 					To install an extension in development from the current directory, use "." as the
```

Comment:
We should probably update this, too

```suggestion
					To install an extension in development from the current directory, use %[1]s.%[1]s as the
```